{"level":"info","message":"âœ… Todas las dependencias han sido configuradas exitosamente.","timestamp":"2025-08-06 13:00:11"}
{"level":"info","message":"ðŸš€ VerificaciÃ³n de dependencias exitosa. El contenedor estÃ¡ listo.","timestamp":"2025-08-06 13:00:11"}
{"level":"info","message":"âœ… Servidor corriendo en el puerto 3001","timestamp":"2025-08-06 13:00:13"}
{"level":"info","message":"UserController: Inicio del proceso de creacion de usuario","timestamp":"2025-08-06 13:07:01"}
{"email":"reyescesar0711@gmail.com","lastname":"Reyes Moreno","level":"info","message":"UserController: Datos enviados por el usuario","name":"Cesar","password":"Hola1234%1234","phone":"04242746760","timestamp":"2025-08-06 13:07:01","username":"Careyes.19"}
{"level":"info","message":"UserController: Llamando al servicio UserService.createUser.","timestamp":"2025-08-06 13:07:01"}
{"level":"info","message":"----Iniciando creacion de usuario en el service: UserService----","timestamp":"2025-08-06 13:07:01"}
{"level":"error","message":"Ha ocurrido un error en el service createUser: UserService:  this.userValidator.validateUserUniqueness is not a function","stack":"TypeError: this.userValidator.validateUserUniqueness is not a function\n    at Proxy.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:30:38)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:45:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:26:53)\n    at C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:20:71\n    at new Promise (<anonymous>)\n    at __awaiter (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:16:12)\n    at Proxy.UserService.createUser (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:65:16)\n    at Proxy.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:41:43)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:34:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:15:53)","timestamp":"2025-08-06 13:07:01"}
{"error":{"code":500,"httpStatusCode":500,"name":"UnexpectedError","originalError":{}},"level":"error","message":"DataUserController: Error durante la creacion del usuario","timestamp":"2025-08-06 13:07:01"}
{"level":"error","message":"GLOBAL ERROR HANDLER: Error capturado: Ha ocurrido un error inesperado: this.userValidator.validateUserUniqueness is not a function. Intente nuevamente.","name":"UnexpectedError","stack":"UnexpectedError: Ha ocurrido un error inesperado: this.userValidator.validateUserUniqueness is not a function. Intente nuevamente.\n    at UnexpectedError.AppError [as constructor] (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\types\\errors.ts:10:14)\n    at new UnexpectedError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\types\\errors.ts:43:14)\n    at handleError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\handleErrors.ts:18:16)\n    at Proxy.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:44:30)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:45:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:26:53)\n    at C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:20:71\n    at new Promise (<anonymous>)\n    at __awaiter (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:16:12)\n    at Proxy.UserService.createUser (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:65:16)","timestamp":"2025-08-06 13:07:01"}
{"level":"info","message":"âœ… Todas las dependencias han sido configuradas exitosamente.","timestamp":"2025-08-06 13:10:56"}
{"level":"info","message":"ðŸš€ VerificaciÃ³n de dependencias exitosa. El contenedor estÃ¡ listo.","timestamp":"2025-08-06 13:10:56"}
{"level":"info","message":"âœ… Servidor corriendo en el puerto 3001","timestamp":"2025-08-06 13:10:58"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateCreateUserData ---","timestamp":"2025-08-06 13:11:03"}
{"level":"info","message":"ValidaciÃ³n de Zod para la creacion de usuario SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 13:11:03"}
{"level":"info","message":"UserController: Inicio del proceso de creacion de usuario","timestamp":"2025-08-06 13:11:03"}
{"email":"reyescesar0711@gmail.com","lastname":"Reyes Moreno","level":"info","message":"UserController: Datos enviados por el usuario","name":"Cesar","password":"Hola1234%1234","phone":"04242746760","timestamp":"2025-08-06 13:11:03","username":"Careyes.19"}
{"level":"info","message":"UserController: Llamando al servicio UserService.createUser.","timestamp":"2025-08-06 13:11:03"}
{"level":"info","message":"----Iniciando creacion de usuario en el service: UserService----","timestamp":"2025-08-06 13:11:03"}
{"level":"error","message":"Ha ocurrido un error en el service createUser: UserService:  this.userValidator.validateUserUniqueness is not a function","stack":"TypeError: this.userValidator.validateUserUniqueness is not a function\n    at Proxy.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:30:38)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:45:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:26:53)\n    at C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:20:71\n    at new Promise (<anonymous>)\n    at __awaiter (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:16:12)\n    at Proxy.UserService.createUser (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:65:16)\n    at Proxy.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:41:43)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:34:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:15:53)","timestamp":"2025-08-06 13:11:03"}
{"error":{"code":500,"httpStatusCode":500,"name":"UnexpectedError","originalError":{}},"level":"error","message":"DataUserController: Error durante la creacion del usuario","timestamp":"2025-08-06 13:11:03"}
{"level":"error","message":"GLOBAL ERROR HANDLER: Error capturado: Ha ocurrido un error inesperado: this.userValidator.validateUserUniqueness is not a function. Intente nuevamente.","name":"UnexpectedError","stack":"UnexpectedError: Ha ocurrido un error inesperado: this.userValidator.validateUserUniqueness is not a function. Intente nuevamente.\n    at UnexpectedError.AppError [as constructor] (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\types\\errors.ts:10:14)\n    at new UnexpectedError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\types\\errors.ts:43:14)\n    at handleError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\handleErrors.ts:18:16)\n    at Proxy.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:44:30)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:45:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:26:53)\n    at C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:20:71\n    at new Promise (<anonymous>)\n    at __awaiter (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:16:12)\n    at Proxy.UserService.createUser (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:65:16)","timestamp":"2025-08-06 13:11:03"}
{"level":"info","message":"âœ… Todas las dependencias han sido configuradas exitosamente.","timestamp":"2025-08-06 13:13:12"}
{"level":"info","message":"ðŸš€ VerificaciÃ³n de dependencias exitosa. El contenedor estÃ¡ listo.","timestamp":"2025-08-06 13:13:12"}
{"level":"info","message":"âœ… Servidor corriendo en el puerto 3001","timestamp":"2025-08-06 13:13:13"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateCreateUserData ---","timestamp":"2025-08-06 13:13:15"}
{"level":"info","message":"ValidaciÃ³n de Zod para la creacion de usuario SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 13:13:15"}
{"level":"info","message":"UserController: Inicio del proceso de creacion de usuario","timestamp":"2025-08-06 13:13:15"}
{"email":"reyescesar0711@gmail.com","lastname":"Reyes Moreno","level":"info","message":"UserController: Datos enviados por el usuario","name":"Cesar","password":"Hola1234%1234","phone":"04242746760","timestamp":"2025-08-06 13:13:15","username":"Careyes.19"}
{"level":"info","message":"UserController: Llamando al servicio UserService.createUser.","timestamp":"2025-08-06 13:13:15"}
{"level":"info","message":"----Iniciando creacion de usuario en el service: UserService----","timestamp":"2025-08-06 13:13:15"}
{"level":"error","message":"Ha ocurrido un error en el service createUser: UserService:  this.userValidator.validateUserUniqueness is not a function","stack":"TypeError: this.userValidator.validateUserUniqueness is not a function\n    at Proxy.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:30:38)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:45:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:26:53)\n    at C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:20:71\n    at new Promise (<anonymous>)\n    at __awaiter (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:16:12)\n    at Proxy.UserService.createUser (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:65:16)\n    at Proxy.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:41:43)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:34:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:15:53)","timestamp":"2025-08-06 13:13:15"}
{"error":{"code":500,"httpStatusCode":500,"name":"UnexpectedError","originalError":{}},"level":"error","message":"DataUserController: Error durante la creacion del usuario","timestamp":"2025-08-06 13:13:15"}
{"level":"error","message":"GLOBAL ERROR HANDLER: Error capturado: Ha ocurrido un error inesperado: this.userValidator.validateUserUniqueness is not a function. Intente nuevamente.","name":"UnexpectedError","stack":"UnexpectedError: Ha ocurrido un error inesperado: this.userValidator.validateUserUniqueness is not a function. Intente nuevamente.\n    at UnexpectedError.AppError [as constructor] (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\types\\errors.ts:10:14)\n    at new UnexpectedError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\types\\errors.ts:43:14)\n    at handleError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\handleErrors.ts:18:16)\n    at Proxy.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:44:30)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:45:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:26:53)\n    at C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:20:71\n    at new Promise (<anonymous>)\n    at __awaiter (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:16:12)\n    at Proxy.UserService.createUser (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:65:16)","timestamp":"2025-08-06 13:13:15"}
{"level":"info","message":"âœ… Todas las dependencias han sido configuradas exitosamente.","timestamp":"2025-08-06 13:18:42"}
{"level":"info","message":"ðŸš€ VerificaciÃ³n de dependencias exitosa. El contenedor estÃ¡ listo.","timestamp":"2025-08-06 13:18:42"}
{"level":"info","message":"âœ… Servidor corriendo en el puerto 3001","timestamp":"2025-08-06 13:18:44"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateCreateUserData ---","timestamp":"2025-08-06 13:18:51"}
{"level":"info","message":"ValidaciÃ³n de Zod para la creacion de usuario SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 13:18:51"}
{"level":"info","message":"UserController: Inicio del proceso de creacion de usuario","timestamp":"2025-08-06 13:18:51"}
{"email":"reyescesar0711@gmail.com","lastname":"Reyes Moreno","level":"info","message":"UserController: Datos enviados por el usuario","name":"Cesar","password":"Hola1234%1234","phone":"04242746760","timestamp":"2025-08-06 13:18:51","username":"Careyes.19"}
{"level":"info","message":"UserController: Llamando al servicio UserService.createUser.","timestamp":"2025-08-06 13:18:51"}
{"level":"info","message":"----Iniciando creacion de usuario en el service: UserService----","timestamp":"2025-08-06 13:18:51"}
{"level":"error","message":"Ha ocurrido un error en el service createUser: UserService:  this.userValidator.validateUserUniqueness is not a function","stack":"TypeError: this.userValidator.validateUserUniqueness is not a function\n    at Proxy.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:30:38)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:45:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:26:53)\n    at C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:20:71\n    at new Promise (<anonymous>)\n    at __awaiter (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:16:12)\n    at Proxy.UserService.createUser (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:65:16)\n    at Proxy.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:41:43)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:34:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:15:53)","timestamp":"2025-08-06 13:18:51"}
{"error":{"code":500,"httpStatusCode":500,"name":"UnexpectedError","originalError":{}},"level":"error","message":"DataUserController: Error durante la creacion del usuario","timestamp":"2025-08-06 13:18:51"}
{"level":"error","message":"GLOBAL ERROR HANDLER: Error capturado: Ha ocurrido un error inesperado: this.userValidator.validateUserUniqueness is not a function. Intente nuevamente.","name":"UnexpectedError","stack":"UnexpectedError: Ha ocurrido un error inesperado: this.userValidator.validateUserUniqueness is not a function. Intente nuevamente.\n    at UnexpectedError.AppError [as constructor] (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\types\\errors.ts:10:14)\n    at new UnexpectedError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\types\\errors.ts:43:14)\n    at handleError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\handleErrors.ts:18:16)\n    at Proxy.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:44:30)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:45:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:26:53)\n    at C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:20:71\n    at new Promise (<anonymous>)\n    at __awaiter (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:16:12)\n    at Proxy.UserService.createUser (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:65:16)","timestamp":"2025-08-06 13:18:51"}
{"level":"info","message":"âœ… Todas las dependencias han sido configuradas exitosamente.","timestamp":"2025-08-06 13:22:43"}
{"level":"info","message":"ðŸš€ VerificaciÃ³n de dependencias exitosa. El contenedor estÃ¡ listo.","timestamp":"2025-08-06 13:22:43"}
{"level":"info","message":"âœ… Servidor corriendo en el puerto 3001","timestamp":"2025-08-06 13:22:44"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateCreateUserData ---","timestamp":"2025-08-06 13:22:50"}
{"level":"info","message":"ValidaciÃ³n de Zod para la creacion de usuario SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 13:22:50"}
{"level":"info","message":"UserController: Inicio del proceso de creacion de usuario","timestamp":"2025-08-06 13:22:50"}
{"email":"reyescesar0711@gmail.com","lastname":"Reyes Moreno","level":"info","message":"UserController: Datos enviados por el usuario","name":"Cesar","password":"Hola1234%1234","phone":"04242746760","timestamp":"2025-08-06 13:22:50","username":"Careyes.19"}
{"level":"info","message":"UserController: Llamando al servicio UserService.createUser.","timestamp":"2025-08-06 13:22:50"}
{"level":"info","message":"----Iniciando creacion de usuario en el service: UserService----","timestamp":"2025-08-06 13:22:50"}
{"level":"error","message":"Ha ocurrido un error en el service createUser: UserService:  La clase UserValidator usa @Transactional pero no tiene una propiedad 'transactionManager'.","stack":"Error: La clase UserValidator usa @Transactional pero no tiene una propiedad 'transactionManager'.\n    at UserValidator.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:21:23)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:34:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:15:53)\n    at C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:9:71\n    at new Promise (<anonymous>)\n    at __awaiter (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:5:12)\n    at UserValidator.descriptor.value [as validateUserUniqueness] (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\transactional-wrapper.ts:49:20)\n    at Proxy.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:30:38)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:45:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:26:53)","timestamp":"2025-08-06 13:22:50"}
{"error":{"code":500,"httpStatusCode":500,"name":"UnexpectedError","originalError":{}},"level":"error","message":"DataUserController: Error durante la creacion del usuario","timestamp":"2025-08-06 13:22:50"}
{"level":"error","message":"GLOBAL ERROR HANDLER: Error capturado: Ha ocurrido un error inesperado: La clase UserValidator usa @Transactional pero no tiene una propiedad 'transactionManager'.. Intente nuevamente.","name":"UnexpectedError","stack":"UnexpectedError: Ha ocurrido un error inesperado: La clase UserValidator usa @Transactional pero no tiene una propiedad 'transactionManager'.. Intente nuevamente.\n    at UnexpectedError.AppError [as constructor] (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\types\\errors.ts:10:14)\n    at new UnexpectedError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\types\\errors.ts:43:14)\n    at handleError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\handleErrors.ts:18:16)\n    at Proxy.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:44:30)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:45:23)\n    at Object.throw (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:26:53)\n    at rejected (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:18:65)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-08-06 13:22:50"}
{"level":"info","message":"âœ… Todas las dependencias han sido configuradas exitosamente.","timestamp":"2025-08-06 13:23:27"}
{"level":"info","message":"ðŸš€ VerificaciÃ³n de dependencias exitosa. El contenedor estÃ¡ listo.","timestamp":"2025-08-06 13:23:27"}
{"level":"info","message":"âœ… Servidor corriendo en el puerto 3001","timestamp":"2025-08-06 13:23:29"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateCreateUserData ---","timestamp":"2025-08-06 13:23:36"}
{"level":"info","message":"ValidaciÃ³n de Zod para la creacion de usuario SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 13:23:36"}
{"level":"info","message":"UserController: Inicio del proceso de creacion de usuario","timestamp":"2025-08-06 13:23:36"}
{"email":"reyescesar0711@gmail.com","lastname":"Reyes Moreno","level":"info","message":"UserController: Datos enviados por el usuario","name":"Cesar","password":"Hola1234%1234","phone":"04242746760","timestamp":"2025-08-06 13:23:36","username":"Careyes.19"}
{"level":"info","message":"UserController: Llamando al servicio UserService.createUser.","timestamp":"2025-08-06 13:23:36"}
{"level":"info","message":"----Iniciando creacion de usuario en el service: UserService----","timestamp":"2025-08-06 13:23:36"}
{"level":"info","message":"----Se ha validado que el usuario no exista previamente----","timestamp":"2025-08-06 13:23:36"}
{"level":"info","message":"----Se ha validado que el usuario no use campos unicos que posea otro usuario----","timestamp":"2025-08-06 13:23:36"}
{"level":"info","message":"----Operacion existosa: Se ha creado el usuario con exito----","timestamp":"2025-08-06 13:23:36"}
{"level":"info","message":"UserController: Usuario creado de forma exitosa.","timestamp":"2025-08-06 13:23:37"}
{"level":"info","message":"âœ… Todas las dependencias han sido configuradas exitosamente.","timestamp":"2025-08-06 13:25:54"}
{"level":"info","message":"ðŸš€ VerificaciÃ³n de dependencias exitosa. El contenedor estÃ¡ listo.","timestamp":"2025-08-06 13:25:54"}
{"level":"info","message":"âœ… Servidor corriendo en el puerto 3001","timestamp":"2025-08-06 13:25:56"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateCreateUserData ---","timestamp":"2025-08-06 13:25:57"}
{"level":"info","message":"ValidaciÃ³n de Zod para la creacion de usuario SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 13:25:57"}
{"level":"info","message":"UserController: Inicio del proceso de creacion de usuario","timestamp":"2025-08-06 13:25:57"}
{"email":"reyescesar683@gmail.com","lastname":"Reyes Moreno","level":"info","message":"UserController: Datos enviados por el usuario","name":"Cesar","password":"Hola1234%1234","phone":"04242746760","timestamp":"2025-08-06 13:25:57","username":"Careyes.20"}
{"level":"info","message":"UserController: Llamando al servicio UserService.createUser.","timestamp":"2025-08-06 13:25:57"}
{"level":"info","message":"----Iniciando creacion de usuario en el service: UserService----","timestamp":"2025-08-06 13:25:57"}
{"level":"info","message":"----Se ha validado que el usuario no exista previamente----","timestamp":"2025-08-06 13:25:57"}
{"code":1102,"httpStatusCode":409,"level":"error","message":"Ha ocurrido un error en el service createUser: UserService:  El correo y el numero de telefono ya se encuentran registrados, intente con unos distintos","name":"UserUniqueFieldsAlreadyExistError","stack":"UserUniqueFieldsAlreadyExistError: El correo y el numero de telefono ya se encuentran registrados, intente con unos distintos\n    at UserUniqueFieldsAlreadyExistError.AppError [as constructor] (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\types\\errors.ts:10:14)\n    at new UserUniqueFieldsAlreadyExistError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\exceptions\\users\\user.exception.ts:22:14)\n    at UserValidator.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\validators\\userValidator.ts:52:31)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\validators\\userValidator.ts:45:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\validators\\userValidator.ts:26:53)\n    at fulfilled (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\validators\\userValidator.ts:17:58)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-08-06 13:25:57"}
{"error":{"code":1102,"httpStatusCode":409,"name":"UserUniqueFieldsAlreadyExistError"},"level":"error","message":"DataUserController: Error durante la creacion del usuario","timestamp":"2025-08-06 13:25:57"}
{"level":"error","message":"GLOBAL ERROR HANDLER: Error capturado: El correo y el numero de telefono ya se encuentran registrados, intente con unos distintos","name":"UserUniqueFieldsAlreadyExistError","stack":"UserUniqueFieldsAlreadyExistError: El correo y el numero de telefono ya se encuentran registrados, intente con unos distintos\n    at UserUniqueFieldsAlreadyExistError.AppError [as constructor] (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\types\\errors.ts:10:14)\n    at new UserUniqueFieldsAlreadyExistError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\exceptions\\users\\user.exception.ts:22:14)\n    at UserValidator.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\validators\\userValidator.ts:52:31)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\validators\\userValidator.ts:45:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\validators\\userValidator.ts:26:53)\n    at fulfilled (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\validators\\userValidator.ts:17:58)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-08-06 13:25:57"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateCreateUserData ---","timestamp":"2025-08-06 13:26:06"}
{"level":"info","message":"ValidaciÃ³n de Zod para la creacion de usuario SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 13:26:06"}
{"level":"info","message":"UserController: Inicio del proceso de creacion de usuario","timestamp":"2025-08-06 13:26:06"}
{"email":"reyescesar683@gmail.com","lastname":"Reyes Moreno","level":"info","message":"UserController: Datos enviados por el usuario","name":"Cesar","password":"Hola1234%1234","phone":"04242746761","timestamp":"2025-08-06 13:26:06","username":"Careyes.20"}
{"level":"info","message":"UserController: Llamando al servicio UserService.createUser.","timestamp":"2025-08-06 13:26:06"}
{"level":"info","message":"----Iniciando creacion de usuario en el service: UserService----","timestamp":"2025-08-06 13:26:06"}
{"level":"info","message":"----Se ha validado que el usuario no exista previamente----","timestamp":"2025-08-06 13:26:06"}
{"level":"info","message":"----Se ha validado que el usuario no use campos unicos que posea otro usuario----","timestamp":"2025-08-06 13:26:06"}
{"level":"info","message":"----Operacion existosa: Se ha creado el usuario con exito----","timestamp":"2025-08-06 13:26:06"}
{"level":"info","message":"UserController: Usuario creado de forma exitosa.","timestamp":"2025-08-06 13:26:06"}
{"level":"info","message":"âœ… Todas las dependencias han sido configuradas exitosamente.","timestamp":"2025-08-06 13:26:34"}
{"level":"info","message":"ðŸš€ VerificaciÃ³n de dependencias exitosa. El contenedor estÃ¡ listo.","timestamp":"2025-08-06 13:26:34"}
{"level":"info","message":"âœ… Servidor corriendo en el puerto 3001","timestamp":"2025-08-06 13:26:36"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateCreateUserData ---","timestamp":"2025-08-06 13:26:51"}
{"level":"info","message":"ValidaciÃ³n de Zod para la creacion de usuario SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 13:26:51"}
{"level":"info","message":"UserController: Inicio del proceso de creacion de usuario","timestamp":"2025-08-06 13:26:51"}
{"email":"cesar.rm199928@gmail.com","lastname":"Reyes Moreno","level":"info","message":"UserController: Datos enviados por el usuario","name":"Cesar","password":"Hola1234%1234","phone":"04242746762","timestamp":"2025-08-06 13:26:51","username":"Careyes.21"}
{"level":"info","message":"UserController: Llamando al servicio UserService.createUser.","timestamp":"2025-08-06 13:26:51"}
{"level":"info","message":"----Iniciando creacion de usuario en el service: UserService----","timestamp":"2025-08-06 13:26:51"}
{"level":"info","message":"----Se ha validado que el usuario no exista previamente----","timestamp":"2025-08-06 13:26:51"}
{"level":"info","message":"----Se ha validado que el usuario no use campos unicos que posea otro usuario----","timestamp":"2025-08-06 13:26:52"}
{"level":"info","message":"----Operacion existosa: Se ha creado el usuario con exito----","timestamp":"2025-08-06 13:26:52"}
{"level":"info","message":"UserController: Usuario creado de forma exitosa.","timestamp":"2025-08-06 13:26:52"}
{"level":"info","message":"âœ… Todas las dependencias han sido configuradas exitosamente.","timestamp":"2025-08-06 13:28:40"}
{"level":"info","message":"ðŸš€ VerificaciÃ³n de dependencias exitosa. El contenedor estÃ¡ listo.","timestamp":"2025-08-06 13:28:40"}
{"level":"info","message":"âœ… Servidor corriendo en el puerto 3001","timestamp":"2025-08-06 13:28:42"}
{"level":"info","message":"âœ… Todas las dependencias han sido configuradas exitosamente.","timestamp":"2025-08-06 13:30:12"}
{"level":"info","message":"ðŸš€ VerificaciÃ³n de dependencias exitosa. El contenedor estÃ¡ listo.","timestamp":"2025-08-06 13:30:12"}
{"level":"info","message":"âœ… Servidor corriendo en el puerto 3001","timestamp":"2025-08-06 13:30:14"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateCreateUserData ---","timestamp":"2025-08-06 13:30:24"}
{"level":"info","message":"ValidaciÃ³n de Zod para la creacion de usuario SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 13:30:24"}
{"level":"info","message":"UserController: Inicio del proceso de creacion de usuario","timestamp":"2025-08-06 13:30:24"}
{"email":"cesar.rm199968@gmail.com","lastname":"Reyes Moreno","level":"info","message":"UserController: Datos enviados por el usuario","name":"Cesar","password":"Hola1234%1234","phone":"04242746763","timestamp":"2025-08-06 13:30:24","username":"Careyes.22"}
{"level":"info","message":"UserController: Llamando al servicio UserService.createUser.","timestamp":"2025-08-06 13:30:24"}
{"level":"info","message":"----Iniciando creacion de usuario en el service: UserService----","timestamp":"2025-08-06 13:30:24"}
{"level":"info","message":"----Se ha validado que el usuario no exista previamente----","timestamp":"2025-08-06 13:30:24"}
{"level":"info","message":"----Se ha validado que el usuario no use campos unicos que posea otro usuario----","timestamp":"2025-08-06 13:30:25"}
{"level":"info","message":"----Operacion existosa: Se ha creado el usuario con exito----","timestamp":"2025-08-06 13:30:25"}
{"level":"info","message":"UserController: Usuario creado de forma exitosa.","timestamp":"2025-08-06 13:30:25"}
{"level":"info","message":"UserController: Inicio del proceso de busqueda de usuarios","timestamp":"2025-08-06 13:36:11"}
{"level":"info","message":"UserController: Llamando al servicio UserService.getUsers.","timestamp":"2025-08-06 13:36:11"}
{"level":"info","message":"----Iniciando la busqueda de los usuarios en el service: UserService----","timestamp":"2025-08-06 13:36:11"}
{"level":"info","message":"----Se ha encontrado el usuario----","timestamp":"2025-08-06 13:36:11"}
{"level":"info","message":"----Operacion existosa: Se ha encontrado los usuarios con exito----","timestamp":"2025-08-06 13:36:11"}
{"level":"info","message":"UserController: Usuario encontrado de forma exitosa.","timestamp":"2025-08-06 13:36:11"}
{"level":"info","message":"âœ… Todas las dependencias han sido configuradas exitosamente.","timestamp":"2025-08-06 13:37:17"}
{"level":"info","message":"ðŸš€ VerificaciÃ³n de dependencias exitosa. El contenedor estÃ¡ listo.","timestamp":"2025-08-06 13:37:17"}
{"level":"info","message":"âœ… Servidor corriendo en el puerto 3001","timestamp":"2025-08-06 13:37:19"}
{"level":"info","message":"UserController: Inicio del proceso de busqueda de usuarios","timestamp":"2025-08-06 13:37:23"}
{"level":"info","message":"UserController: Llamando al servicio UserService.getUsers.","timestamp":"2025-08-06 13:37:23"}
{"level":"info","message":"----Iniciando la busqueda de los usuarios en el service: UserService----","timestamp":"2025-08-06 13:37:23"}
{"level":"info","message":"----Se ha encontrado el usuario----","timestamp":"2025-08-06 13:37:23"}
{"level":"info","message":"----Operacion existosa: Se ha encontrado los usuarios con exito----","timestamp":"2025-08-06 13:37:23"}
{"level":"info","message":"UserController: Usuario encontrado de forma exitosa.","timestamp":"2025-08-06 13:37:23"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateCreateUserData ---","timestamp":"2025-08-06 15:03:33"}
{"level":"info","message":"ValidaciÃ³n de Zod para la creacion de usuario SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 15:03:33"}
{"level":"info","message":"UserController: Inicio del proceso de creacion de usuario","timestamp":"2025-08-06 15:03:33"}
{"email":"carlos.rodriguez@empresa.com","lastname":"Reyes","level":"info","message":"UserController: Datos enviados por el usuario","name":"Cesar","password":"Hola1234%1234","phone":"04245675645","timestamp":"2025-08-06 15:03:33","username":"cesarreyes"}
{"level":"info","message":"UserController: Llamando al servicio UserService.createUser.","timestamp":"2025-08-06 15:03:33"}
{"level":"info","message":"----Iniciando creacion de usuario en el service: UserService----","timestamp":"2025-08-06 15:03:33"}
{"level":"info","message":"----Se ha validado que el usuario no exista previamente----","timestamp":"2025-08-06 15:03:34"}
{"level":"info","message":"----Se ha validado que el usuario no use campos unicos que posea otro usuario----","timestamp":"2025-08-06 15:03:34"}
{"level":"info","message":"----Operacion existosa: Se ha creado el usuario con exito----","timestamp":"2025-08-06 15:03:34"}
{"level":"info","message":"UserController: Usuario creado de forma exitosa.","timestamp":"2025-08-06 15:03:34"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateLoginData ---","timestamp":"2025-08-06 15:09:50"}
{"level":"info","message":"ValidaciÃ³n de Zod para login SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 15:09:50"}
{"level":"info","message":"AuthController: Inicio del proceso de login","timestamp":"2025-08-06 15:09:50"}
{"level":"info","message":"AuthController: Datos enviados por el usuario","password":"Hola1234%1234","timestamp":"2025-08-06 15:09:50","username":"Careyes.19"}
{"level":"info","message":"AuthController: Llamando al servicio AuthService.loginAuthDto.","timestamp":"2025-08-06 15:09:50"}
{"level":"info","message":"----Iniciando la busqueda del usuario en el service: UserService----","timestamp":"2025-08-06 15:09:50"}
{"level":"info","message":"----Se ha encontrado el usuario----","timestamp":"2025-08-06 15:09:50"}
{"level":"info","message":"----Operacion existosa: Se ha encontrado el usuario con exito----","timestamp":"2025-08-06 15:09:50"}
{"error":{"code":500,"httpStatusCode":500,"name":"UnexpectedError","originalError":{}},"level":"error","message":"AuthController: Error durante el login del usuario","timestamp":"2025-08-06 15:09:51"}
{"level":"error","message":"GLOBAL ERROR HANDLER: Error capturado: Ha ocurrido un error inesperado: Cannot inject the dependency at position #0 of \"TokenService\" constructor. Reason:\n    Attempted to resolve unregistered dependency token: \"Symbol(JWT_SECRET_TOKEN)\". Intente nuevamente.","name":"UnexpectedError","stack":"UnexpectedError: Ha ocurrido un error inesperado: Cannot inject the dependency at position #0 of \"TokenService\" constructor. Reason:\n    Attempted to resolve unregistered dependency token: \"Symbol(JWT_SECRET_TOKEN)\". Intente nuevamente.\n    at UnexpectedError.AppError [as constructor] (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\types\\errors.ts:10:14)\n    at new UnexpectedError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\types\\errors.ts:43:14)\n    at handleError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\handleErrors.ts:18:16)\n    at AuthService.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\auth\\Auth.service.ts:37:30)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\auth\\Auth.service.ts:45:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\auth\\Auth.service.ts:26:53)\n    at fulfilled (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\auth\\Auth.service.ts:17:58)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-08-06 15:09:51"}
{"level":"info","message":"âœ… Todas las dependencias han sido configuradas exitosamente.","timestamp":"2025-08-06 16:25:16"}
{"level":"info","message":"ðŸš€ VerificaciÃ³n de dependencias exitosa. El contenedor estÃ¡ listo.","timestamp":"2025-08-06 16:25:16"}
{"level":"info","message":"âœ… Servidor corriendo en el puerto 3001","timestamp":"2025-08-06 16:25:18"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateLoginData ---","timestamp":"2025-08-06 16:25:23"}
{"level":"info","message":"ValidaciÃ³n de Zod para login SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 16:25:23"}
{"level":"info","message":"AuthController: Inicio del proceso de login","timestamp":"2025-08-06 16:25:23"}
{"level":"info","message":"AuthController: Datos enviados por el usuario","password":"Hola1234%1234","timestamp":"2025-08-06 16:25:23","username":"Careyes.19"}
{"level":"info","message":"AuthController: Llamando al servicio AuthService.loginAuthDto.","timestamp":"2025-08-06 16:25:23"}
{"level":"info","message":"----Iniciando la busqueda del usuario en el service: UserService----","timestamp":"2025-08-06 16:25:23"}
{"level":"info","message":"----Se ha encontrado el usuario----","timestamp":"2025-08-06 16:25:24"}
{"level":"info","message":"----Operacion existosa: Se ha encontrado el usuario con exito----","timestamp":"2025-08-06 16:25:24"}
{"error":{"code":500,"httpStatusCode":500,"name":"UnexpectedError","originalError":{}},"level":"error","message":"AuthController: Error durante el login del usuario","timestamp":"2025-08-06 16:25:24"}
{"level":"error","message":"GLOBAL ERROR HANDLER: Error capturado: Ha ocurrido un error inesperado: Cannot inject the dependency at position #0 of \"TokenService\" constructor. Reason:\n    TypeInfo not known for \"undefined\". Intente nuevamente.","name":"UnexpectedError","stack":"UnexpectedError: Ha ocurrido un error inesperado: Cannot inject the dependency at position #0 of \"TokenService\" constructor. Reason:\n    TypeInfo not known for \"undefined\". Intente nuevamente.\n    at UnexpectedError.AppError [as constructor] (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\types\\errors.ts:10:14)\n    at new UnexpectedError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\types\\errors.ts:43:14)\n    at handleError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\handleErrors.ts:18:16)\n    at AuthService.<anonymous> (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\auth\\Auth.service.ts:37:30)\n    at step (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\auth\\Auth.service.ts:45:23)\n    at Object.next (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\auth\\Auth.service.ts:26:53)\n    at fulfilled (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\auth\\Auth.service.ts:17:58)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-08-06 16:25:24"}
{"level":"info","message":"âœ… Todas las dependencias han sido configuradas exitosamente.","timestamp":"2025-08-06 16:30:17"}
{"level":"info","message":"ðŸš€ VerificaciÃ³n de dependencias exitosa. El contenedor estÃ¡ listo.","timestamp":"2025-08-06 16:30:17"}
{"level":"info","message":"âœ… Servidor corriendo en el puerto 3001","timestamp":"2025-08-06 16:30:19"}
{"level":"info","message":"âœ… Todas las dependencias han sido configuradas exitosamente.","timestamp":"2025-08-06 16:31:16"}
{"level":"info","message":"ðŸš€ VerificaciÃ³n de dependencias exitosa. El contenedor estÃ¡ listo.","timestamp":"2025-08-06 16:31:16"}
{"level":"info","message":"âœ… Servidor corriendo en el puerto 3001","timestamp":"2025-08-06 16:31:17"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateLoginData ---","timestamp":"2025-08-06 16:31:22"}
{"level":"info","message":"ValidaciÃ³n de Zod para login SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 16:31:22"}
{"level":"info","message":"AuthController: Inicio del proceso de login","timestamp":"2025-08-06 16:31:22"}
{"level":"info","message":"AuthController: Datos enviados por el usuario","password":"Hola1234%1234","timestamp":"2025-08-06 16:31:22","username":"Careyes.19"}
{"level":"info","message":"AuthController: Llamando al servicio AuthService.loginAuthDto.","timestamp":"2025-08-06 16:31:22"}
{"level":"info","message":"----Iniciando la busqueda del usuario en el service: UserService----","timestamp":"2025-08-06 16:31:22"}
{"level":"info","message":"----Se ha encontrado el usuario----","timestamp":"2025-08-06 16:31:22"}
{"level":"info","message":"----Operacion existosa: Se ha encontrado el usuario con exito----","timestamp":"2025-08-06 16:31:22"}
{"error":{"code":500,"httpStatusCode":500,"name":"UnexpectedError","originalError":{}},"level":"error","message":"AuthController: Error durante el login del usuario","timestamp":"2025-08-06 16:31:22"}
{"level":"error","message":"GLOBAL ERROR HANDLER: Error capturado: Ha ocurrido un error inesperado: Cannot inject the dependency \"secret\" at position #0 of \"TokenService\" constructor. Reason:\n    TypeInfo not known for \"undefined\". Intente nuevamente.","name":"UnexpectedError","stack":"UnexpectedError: Ha ocurrido un error inesperado: Cannot inject the dependency \"secret\" at position #0 of \"TokenService\" constructor. Reason:\n    TypeInfo not known for \"undefined\". Intente nuevamente.\n    at handleError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\handleErrors.ts:18:16)\n    at AuthService.loginUser (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\auth\\Auth.service.ts:37:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async AuthController.loginUser (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\controllers\\auth\\auth.controller.ts:32:28)","timestamp":"2025-08-06 16:31:22"}
{"level":"info","message":"âœ… Todas las dependencias han sido configuradas exitosamente.","timestamp":"2025-08-06 16:32:17"}
{"level":"info","message":"ðŸš€ VerificaciÃ³n de dependencias exitosa. El contenedor estÃ¡ listo.","timestamp":"2025-08-06 16:32:17"}
{"level":"info","message":"âœ… Servidor corriendo en el puerto 3001","timestamp":"2025-08-06 16:32:19"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateLoginData ---","timestamp":"2025-08-06 16:32:22"}
{"level":"info","message":"ValidaciÃ³n de Zod para login SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 16:32:22"}
{"level":"info","message":"AuthController: Inicio del proceso de login","timestamp":"2025-08-06 16:32:22"}
{"level":"info","message":"AuthController: Datos enviados por el usuario","password":"Hola1234%1234","timestamp":"2025-08-06 16:32:22","username":"Careyes.19"}
{"level":"info","message":"AuthController: Llamando al servicio AuthService.loginAuthDto.","timestamp":"2025-08-06 16:32:22"}
{"level":"info","message":"----Iniciando la busqueda del usuario en el service: UserService----","timestamp":"2025-08-06 16:32:22"}
{"level":"info","message":"----Se ha encontrado el usuario----","timestamp":"2025-08-06 16:32:22"}
{"level":"info","message":"----Operacion existosa: Se ha encontrado el usuario con exito----","timestamp":"2025-08-06 16:32:22"}
{"error":{"code":500,"httpStatusCode":500,"name":"UnexpectedError","originalError":{}},"level":"error","message":"AuthController: Error durante el login del usuario","timestamp":"2025-08-06 16:32:22"}
{"level":"error","message":"GLOBAL ERROR HANDLER: Error capturado: Ha ocurrido un error inesperado: Cannot inject the dependency \"secret\" at position #0 of \"TokenService\" constructor. Reason:\n    TypeInfo not known for \"undefined\". Intente nuevamente.","name":"UnexpectedError","stack":"UnexpectedError: Ha ocurrido un error inesperado: Cannot inject the dependency \"secret\" at position #0 of \"TokenService\" constructor. Reason:\n    TypeInfo not known for \"undefined\". Intente nuevamente.\n    at handleError (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\utils\\handleErrors.ts:18:16)\n    at AuthService.loginUser (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\auth\\Auth.service.ts:37:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async AuthController.loginUser (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\controllers\\auth\\auth.controller.ts:32:28)","timestamp":"2025-08-06 16:32:22"}
{"level":"info","message":"âœ… Todas las dependencias han sido configuradas exitosamente.","timestamp":"2025-08-06 16:37:03"}
{"level":"info","message":"ðŸš€ VerificaciÃ³n de dependencias exitosa. El contenedor estÃ¡ listo.","timestamp":"2025-08-06 16:37:03"}
{"level":"info","message":"âœ… Servidor corriendo en el puerto 3001","timestamp":"2025-08-06 16:37:04"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateLoginData ---","timestamp":"2025-08-06 16:37:11"}
{"level":"info","message":"ValidaciÃ³n de Zod para login SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 16:37:11"}
{"level":"info","message":"AuthController: Inicio del proceso de login","timestamp":"2025-08-06 16:37:11"}
{"level":"info","message":"AuthController: Datos enviados por el usuario","password":"Hola1234%1234","timestamp":"2025-08-06 16:37:11","username":"Careyes.19"}
{"level":"info","message":"AuthController: Llamando al servicio AuthService.loginAuthDto.","timestamp":"2025-08-06 16:37:11"}
{"level":"info","message":"----Iniciando la busqueda del usuario en el service: UserService----","timestamp":"2025-08-06 16:37:11"}
{"level":"info","message":"----Se ha encontrado el usuario----","timestamp":"2025-08-06 16:37:11"}
{"level":"info","message":"----Operacion existosa: Se ha encontrado el usuario con exito----","timestamp":"2025-08-06 16:37:11"}
{"level":"info","message":"AuthController: login creado de forma exitosa.","timestamp":"2025-08-06 16:37:11"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateLoginData ---","timestamp":"2025-08-06 16:37:46"}
{"level":"info","message":"ValidaciÃ³n de Zod para login SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 16:37:46"}
{"level":"info","message":"AuthController: Inicio del proceso de login","timestamp":"2025-08-06 16:37:46"}
{"level":"info","message":"AuthController: Datos enviados por el usuario","password":"Hola1234%1234","timestamp":"2025-08-06 16:37:46","username":"Careyes.19"}
{"level":"info","message":"AuthController: Llamando al servicio AuthService.loginAuthDto.","timestamp":"2025-08-06 16:37:46"}
{"level":"info","message":"----Iniciando la busqueda del usuario en el service: UserService----","timestamp":"2025-08-06 16:37:46"}
{"level":"info","message":"----Se ha encontrado el usuario----","timestamp":"2025-08-06 16:37:46"}
{"level":"info","message":"----Operacion existosa: Se ha encontrado el usuario con exito----","timestamp":"2025-08-06 16:37:46"}
{"level":"info","message":"AuthController: login creado de forma exitosa.","timestamp":"2025-08-06 16:37:46"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateLoginData ---","timestamp":"2025-08-06 16:38:17"}
{"level":"info","message":"ValidaciÃ³n de Zod para login SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 16:38:17"}
{"level":"info","message":"AuthController: Inicio del proceso de login","timestamp":"2025-08-06 16:38:17"}
{"level":"info","message":"AuthController: Datos enviados por el usuario","password":"Hola1234$1234","timestamp":"2025-08-06 16:38:17","username":"Careyes.191"}
{"level":"info","message":"AuthController: Llamando al servicio AuthService.loginAuthDto.","timestamp":"2025-08-06 16:38:17"}
{"level":"info","message":"----Iniciando la busqueda del usuario en el service: UserService----","timestamp":"2025-08-06 16:38:17"}
{"level":"info","message":"----Se ha encontrado el usuario----","timestamp":"2025-08-06 16:38:17"}
{"code":1100,"httpStatusCode":404,"level":"error","message":"Ha ocurrido un error en el service findUser: UserService:  El usuario no existe","name":"UserNotFoundError","stack":"UserNotFoundError: El usuario no existe\n    at UserValidator.validateUserExists (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\validators\\userValidator.ts:19:25)\n    at UserService.findUserByUsername (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:85:32)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async TransactionManager.executeTransaction (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\database\\transactionManager.ts:19:28)\n    at async AuthService.loginUser (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\auth\\Auth.service.ts:21:26)\n    at async AuthController.loginUser (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\controllers\\auth\\auth.controller.ts:32:28)","timestamp":"2025-08-06 16:38:17"}
{"error":{"code":1100,"httpStatusCode":404,"name":"UserNotFoundError"},"level":"error","message":"AuthController: Error durante el login del usuario","timestamp":"2025-08-06 16:38:17"}
{"level":"error","message":"GLOBAL ERROR HANDLER: Error capturado: El usuario no existe","name":"UserNotFoundError","stack":"UserNotFoundError: El usuario no existe\n    at UserValidator.validateUserExists (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\validators\\userValidator.ts:19:25)\n    at UserService.findUserByUsername (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\user\\User.service.ts:85:32)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async TransactionManager.executeTransaction (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\core\\database\\transactionManager.ts:19:28)\n    at async AuthService.loginUser (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\services\\auth\\Auth.service.ts:21:26)\n    at async AuthController.loginUser (C:\\Users\\PC\\Documents\\gestor-tareas-backend\\src\\controllers\\auth\\auth.controller.ts:32:28)","timestamp":"2025-08-06 16:38:17"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateLoginData ---","timestamp":"2025-08-06 16:41:38"}
{"level":"info","message":"ValidaciÃ³n de Zod para login SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 16:41:38"}
{"level":"info","message":"AuthController: Inicio del proceso de login","timestamp":"2025-08-06 16:41:38"}
{"level":"info","message":"AuthController: Datos enviados por el usuario","password":"Hola1234%1234","timestamp":"2025-08-06 16:41:38","username":"Careyes.19"}
{"level":"info","message":"AuthController: Llamando al servicio AuthService.loginAuthDto.","timestamp":"2025-08-06 16:41:38"}
{"level":"info","message":"----Iniciando la busqueda del usuario en el service: UserService----","timestamp":"2025-08-06 16:41:38"}
{"level":"info","message":"----Se ha encontrado el usuario----","timestamp":"2025-08-06 16:41:38"}
{"level":"info","message":"----Operacion existosa: Se ha encontrado el usuario con exito----","timestamp":"2025-08-06 16:41:38"}
{"level":"info","message":"AuthController: login creado de forma exitosa.","timestamp":"2025-08-06 16:41:38"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateLoginData ---","timestamp":"2025-08-06 16:43:15"}
{"level":"info","message":"ValidaciÃ³n de Zod para login SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 16:43:15"}
{"level":"info","message":"AuthController: Inicio del proceso de login","timestamp":"2025-08-06 16:43:15"}
{"level":"info","message":"AuthController: Datos enviados por el usuario","password":"Hola1234%1234","timestamp":"2025-08-06 16:43:15","username":"Careyes.19"}
{"level":"info","message":"AuthController: Llamando al servicio AuthService.loginAuthDto.","timestamp":"2025-08-06 16:43:15"}
{"level":"info","message":"----Iniciando la busqueda del usuario en el service: UserService----","timestamp":"2025-08-06 16:43:15"}
{"level":"info","message":"----Se ha encontrado el usuario----","timestamp":"2025-08-06 16:43:15"}
{"level":"info","message":"----Operacion existosa: Se ha encontrado el usuario con exito----","timestamp":"2025-08-06 16:43:15"}
{"level":"info","message":"AuthController: login creado de forma exitosa.","timestamp":"2025-08-06 16:43:15"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateLoginData ---","timestamp":"2025-08-06 16:50:57"}
{"level":"info","message":"ValidaciÃ³n de Zod para login SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 16:50:57"}
{"level":"info","message":"AuthController: Inicio del proceso de login","timestamp":"2025-08-06 16:50:57"}
{"level":"info","message":"AuthController: Datos enviados por el usuario","password":"Hola1234%1234","timestamp":"2025-08-06 16:50:57","username":"Careyes.19"}
{"level":"info","message":"AuthController: Llamando al servicio AuthService.loginAuthDto.","timestamp":"2025-08-06 16:50:57"}
{"level":"info","message":"----Iniciando la busqueda del usuario en el service: UserService----","timestamp":"2025-08-06 16:50:57"}
{"level":"info","message":"----Se ha encontrado el usuario----","timestamp":"2025-08-06 16:50:57"}
{"level":"info","message":"----Operacion existosa: Se ha encontrado el usuario con exito----","timestamp":"2025-08-06 16:50:57"}
{"level":"info","message":"AuthController: login creado de forma exitosa.","timestamp":"2025-08-06 16:50:57"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateLoginData ---","timestamp":"2025-08-06 16:51:12"}
{"level":"info","message":"ValidaciÃ³n de Zod para login SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 16:51:12"}
{"level":"info","message":"AuthController: Inicio del proceso de login","timestamp":"2025-08-06 16:51:12"}
{"level":"info","message":"AuthController: Datos enviados por el usuario","password":"Hola1234%1234","timestamp":"2025-08-06 16:51:12","username":"Careyes.19"}
{"level":"info","message":"AuthController: Llamando al servicio AuthService.loginAuthDto.","timestamp":"2025-08-06 16:51:12"}
{"level":"info","message":"----Iniciando la busqueda del usuario en el service: UserService----","timestamp":"2025-08-06 16:51:12"}
{"level":"info","message":"----Se ha encontrado el usuario----","timestamp":"2025-08-06 16:51:12"}
{"level":"info","message":"----Operacion existosa: Se ha encontrado el usuario con exito----","timestamp":"2025-08-06 16:51:12"}
{"level":"info","message":"AuthController: login creado de forma exitosa.","timestamp":"2025-08-06 16:51:12"}
{"level":"info","message":"--- MIDDLEWARE: Ejecutando validateLoginData ---","timestamp":"2025-08-06 16:52:42"}
{"level":"info","message":"ValidaciÃ³n de Zod para login SUPERADA. Pasando al controlador.","timestamp":"2025-08-06 16:52:42"}
{"level":"info","message":"AuthController: Inicio del proceso de login","timestamp":"2025-08-06 16:52:42"}
{"level":"info","message":"AuthController: Datos enviados por el usuario","password":"Hola1234%1234","timestamp":"2025-08-06 16:52:42","username":"Careyes.19"}
{"level":"info","message":"AuthController: Llamando al servicio AuthService.loginAuthDto.","timestamp":"2025-08-06 16:52:42"}
{"level":"info","message":"----Iniciando la busqueda del usuario en el service: UserService----","timestamp":"2025-08-06 16:52:42"}
{"level":"info","message":"----Se ha encontrado el usuario----","timestamp":"2025-08-06 16:52:42"}
{"level":"info","message":"----Operacion existosa: Se ha encontrado el usuario con exito----","timestamp":"2025-08-06 16:52:42"}
{"level":"info","message":"AuthController: login creado de forma exitosa.","timestamp":"2025-08-06 16:52:42"}
